src/main/scala/org/hammerlab/guacamole/readsets/rdd/PartitionedRegionsArgs.scala

```diff
@@ -0,0 +1,61 @@
+package org.hammerlab.guacamole.readsets.rdd
+
+import org.apache.hadoop.fs.Path
+import org.apache.spark.rdd.RDD
+import org.hammerlab.guacamole.loci.partitioning.{LociPartitionerArgs, LociPartitioning}
+import org.hammerlab.magic.args4j.StringOptionHandler
+import org.kohsuke.args4j.spi.BooleanOptionHandler
+import org.kohsuke.args4j.{Option => Args4JOption}
+
+/**
+ * Command-line arguments related to partitioning regions according to a [[LociPartitioning]].
+ *
+ * Allows for configuring how/whether to persist the computed [[PartitionedRegions]] (its [[RDD]] of regions and/or its
+ * [[LociPartitioning]]).
+ */
+trait PartitionedRegionsArgs extends LociPartitionerArgs {
+  @Args4JOption(
```

Comment:
Not obvious to me what's going with `--partitioning-dir` vs. `--partitioned-reads`.  A header comment explaining what the difference between these two options is would be useful